Source/Core/Common/SysConf.h

```diff
@@ -74,7 +74,12 @@ struct SSysConfEntry
 class SysConf
 {
 public:
-  SysConf();
+  enum class WiiRootType
```

Comment:
We already have an enum like this: https://github.com/dolphin-emu/dolphin/blob/3cfc49a613a0f8e137c251694c97f0f3589f3216/Source/Core/Common/NandPaths.h#L21